server/commands.lua

```diff
@@ -253,6 +253,58 @@ lib.addCommand('setjob', {
     end
 end)
 
+
+--- ADMIN COMMAND
+
+
+lib.addCommand('addjob', {
+    help = 'Add Job to Player',
+    params = {
+        {
+            name = 'target',
+            type = 'playerId',
+            help = 'Target player\'s server id',
+        },
+        {
+            name = 'jobname',
+            type = 'string',
+            help = 'Job Code',
+        },
+        {
+            name = 'grade',
+            type = 'number',
+            help = 'Job Grade',
+        },
+    },
+    restricted = 'group.admin'
+}, function(source, args)
+    local TargetPlayer = exports.qbx_core:GetPlayer(args.target)
+    local citizenid = TargetPlayer.PlayerData.citizenid
+    exports.qbx_core:AddPlayerToJob(citizenid, args.jobname, args.grade)
```

Comment:
```suggestion
    local player = GetPlayer(tonumber(args[locale("command.addjob.params.id.name")]) --[[@as number]])
    if not player then
        Notify(source, locale('error.not_online'), 'error')
        return
    end
    if args[locale("command.addjob.params.grade.name")] then
        AddPlayerToJob(player.PlayerData.citizenid, tostring(args[locale("command.addjob.params.job.name")]), tonumber(args[locale("command.addjob.params.grade.name")]) --[[@as number]])
    else
        AddPlayerToJob(player.PlayerData.citizenid, tostring(args[locale("command.addjob.params.job.name")]), 0)
    end
```

As we are in the core we don't need to call the exports.